cloud/deployment/deployment.go

```diff
@@ -380,6 +387,7 @@ func Create(name, workspaceID, description, clusterID, runtimeVersion, dagDeploy
 				IsCicdEnforced:       isCicdEnforced,
 				IsDagDeployEnabled:   dagDeployEnabled,
 				IsHighAvailability:   highAvailabilityValue,
+				IsDevelopmentMode:    &developmentModeValue,
```

Comment:
same here, should this be `&` ?

Reply:
https://github.com/astronomer/astro-cli/pull/1537#discussion_r1490495672